planning/behavior_path_planner/docs/behavior_path_planner_manager_design.md

```diff
@@ -0,0 +1,487 @@
+# Manager design
+
+## Purpose / Role
+
+The manager launches and executes scene modules in `behavior_path_planner` depending on the use case, and has been developed to achieve following features:
+
+- Multiple modules can run simultaneously in series in order to achieve more complex use cases.
+- Don't use external library (e.g. BehaviorTree.cpp) for flexible development.
+
+Support status:
+
+| Name                     | Simple exclusive execution | Advanced simultaneous execution |
+| :----------------------- | :------------------------: | :-----------------------------: |
+| Avoidance                |     :heavy_check_mark:     |       :heavy_check_mark:        |
+| Avoidance By Lane Change |     :heavy_check_mark:     |    :heavy_multiplication_x:     |
+| Lane Change              |     :heavy_check_mark:     |       :heavy_check_mark:        |
+| External Lane Change     |     :heavy_check_mark:     |    :heavy_multiplication_x:     |
+| Pull Over                |     :heavy_check_mark:     |    :heavy_multiplication_x:     |
+| Pull Out                 |     :heavy_check_mark:     |    :heavy_multiplication_x:     |
+| Side Shift               |     :heavy_check_mark:     |    :heavy_multiplication_x:     |
+
+Click [here](../README.md) for supported scene modules.
+
+!!! warning
+
+    It is still under development and some functions may be unstable.
+
+## Overview
+
+The manager is the implementation part of the core of `behavior_path_planner`. It outputs path based on the latest data.
```

Comment:
```suggestion
The manager is the core part of `behavior_path_planner` implementation. It outputs path based on the latest data.
```

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/3493/commits/7d79ac77e442933f2cfcd7b49ee57c030f71b189.